src/pudl/transform/eia.py

```diff
@@ -661,11 +658,8 @@ def harvest_entity_tables(  # noqa: C901
     logger.info(f"Average consistency of static {entity.value} values is {mcs:.2%}")
 
     # Apply standard PUDL data types to the new entity tables:
-    pkg = Package.from_resource_ids()
-    entity_res = pkg.get_resource(f"core_eia__entity_{entity.value}")
-    entity_df = apply_pudl_dtypes(entity_df, group="eia").pipe(entity_res.encode)
-    annual_res = pkg.get_resource(f"core_eia860__scd_{entity.value}")
-    annual_df = apply_pudl_dtypes(annual_df, group="eia").pipe(annual_res.encode)
+    entity_df = PUDL_PKG.encode(apply_pudl_dtypes(entity_df, group="eia"))
+    annual_df = PUDL_PKG.encode(apply_pudl_dtypes(annual_df, group="eia"))
```

Comment:
this seems fine to do but also in theory wouldn't this be duplicative if all of the inputs to the harvesting process are already encoded?

Reply:
Definitely. Where's the best place to inject the encoding for all the tables? We don't have a `transform_end()` step here like we do for FERC 1, but it would be nice to ensure that every table gets encoded by default before it comes out of the `_core_eia*` step.